packages/app/pages/AllProjectsPage/AllProjectsPage.tsx

```diff
@@ -19,43 +21,140 @@ interface Props {
   children: Object
 }
 
+/**
+ * Row of buttons to filter which user agent types should be displayed
+ */
+class NavBar extends React.Component {
+
+  readonly navButtons = ["All", "Cooperative", "Projects", "Organizations", "Groups"]
+
+  private currentTheme;
+
+  constructor(private props) {
+    super(props)
+    this.state = {
+      activeButton: "All",
+    };
+
+    this.currentTheme = themeable(this.props.theme)
+  }
+
+  /**
+   * Handler function for when buttons are pressed.
+   * TODO apply the filter to the user agents
+   * @param buttonName the text of the button that was clicked
+   */
+  handleClick = (buttonName) => {
+    this.setState({ activeButton: buttonName })
+    this.forceUpdate(() => console.log('Finished Updating'))
+    // this.state.activeButton  = buttonName
+    console.log("Clicked", buttonName)
+    // TODO refresh the list and filter
```

Comment:
Try to add when TODOs need to be completed by.

For example
TODO before implementing material add
or something along those lines so someone implementing the affected functionality in the future can do a quick scan for todo's